packages/flutter/lib/src/widgets/scroll_notification.dart

```diff
@@ -140,13 +208,24 @@ class OverscrollNotification extends ScrollNotification {
   }
 }
 
+/// A notification that [Scrollable] widget has stopped scrolling.
+///
+/// See also:
+///
+///  * [ScrollStartNotification], which indicates that scrolling has started.
+///  * [ScrollNotification], which describes the notification lifecycle.
 class ScrollEndNotification extends ScrollNotification {
+  /// Creates a notification that [Scrollable] widget has stopped scrolling.
   ScrollEndNotification({
     @required ScrollMetrics metrics,
     @required BuildContext context,
     this.dragDetails,
   }) : super(metrics: metrics, context: context);
 
+  /// If the [Scrollable] stopped scrolling because of a drag, the details about
```

Comment:
It would be helpful to explain when the dragDetails are non-null, i.e. the drag gesture ended with a velocity that was to small to trigger a ballistic scroll.